app/build.gradle

```diff
@@ -170,7 +170,6 @@ dependencies {
     implementation "com.github.bumptech.glide:glide:$rootProject.glideVersion"
     implementation "com.github.bumptech.glide:gifencoder-integration:$rootProject.glideVersion"
     implementation 'com.yalantis:ucrop:1.5.0'
-    implementation 'de.psdev.licensesdialog:licensesdialog:1.8.0'
```

Comment:
Any particular reason for removing this? It can be kept in the `Special thanks` part since we do use a lot of external dependencies.